cookbook/case_studies/feature_engineering/sqlite_datacleaning/README.rst

```diff
@@ -0,0 +1,63 @@
+Data Cleaning
+-------------
+Feature Engineering off-late has become one of the most prominent topics in Machine Learning. 
+It is the process of transforming raw data into features that better represent the underlying problem to the predictive models, resulting in improved model accuracy on unseen data.
+
+This tutorial will implement data cleaning of SQLite3 data, which does both data imputation and univariate feature selection. These are so-called feature engineering techniques.
```

Comment:
Can we specifically explain that we used sqlite3 as an example, but these methods are applicable to any query engine?

Also at a high level we should explain what we are trying to demonstrate 

1. Feature engineering where source data is from sql like data sources 
2. Procreated Feature transforms
3. Ability to create a low code - platform using this